templates/steering-display/platformio.ini

```diff
@@ -26,8 +26,8 @@ board_build.f_cpu = 180000000L
 monitor_speed = 115200
 debug_tool = blackmagic
 upload_protocol = blackmagic
-upload_port = COM4 ; These ports will vary by host computer and platform
```

Comment:
Wonder if there is an easy way to prevent these lines from constantly being changed since ports are not static even on the same machine. Maybe we can just leave it commented by default?